tests/_utils/index.js

```diff
@@ -2,13 +2,11 @@
  * @fileoverview Utilities used in tests
  */
 
-"use strict";
-
 //-----------------------------------------------------------------------------
 // Requirements
 //-----------------------------------------------------------------------------
 
-const { createTeardown, addFile } = require("fs-teardown");
+import { createTeardown, addFile } from "fs-teardown";
```

Comment:
```suggestion
import fsTeardown from "fs-teardown";

const { createTeardown, addFile } = fsTeardown;
```

Same as https://github.com/eslint/eslint-scope/pull/71#discussion_r663382416